src/Surfnet/ServiceProviderDashboard/Application/Assembler/ServiceStatusAssembler.php

```diff
@@ -78,30 +80,22 @@ class ServiceStatusAssembler
 
     /**
      * The mapping of the legend
-     *
-     * @var array
      */
-    private $legend = [
+    private array $legend = [
         self::SERVICE_STATUS_SUCCESS => '#67a979',
         self::SERVICE_STATUS_IN_PROGRESS => '#f6aa61',
         self::SERVICE_STATUS_INFO => '#d1d2d6',
     ];
```

Comment:
Not saying you should. But now that we use PHP82, we could use Enums here. Consider this a mental note for my future self

Reply:
Yes we could, but for now out odf scope